Configure.pl

```diff
@@ -53,10 +53,15 @@
 
     unless (defined $options{prefix}) {
         my $default = defined($options{sysroot}) ? '/usr' : File::Spec->catdir(getcwd, 'install');
-        print "ATTENTION: no --prefix supplied, building and installing to $default\n";
+        # print "ATTENTION: no --prefix supplied, building and installing to $default\n";
         $options{prefix} = $default;
     }
     $options{prefix} = File::Spec->rel2abs($options{prefix});
+    unless (defined $options{libdir}) {
+        my $default = File::Spec->catdir($options{prefix}, 'lib');
```

Comment:
Defaulting to $prefix/lib breaks compatibility with existing installations forcing our users to re-install all modules. We should not do that without an excellent reason.